src/main/webapp/css/loginPage.css

```diff
@@ -0,0 +1,41 @@
+    .login-form {
```

Comment:
This is acceptable, but I recommend to have a ticket to define constants for common styles. For e.g: font-size,

Reply:
I will change my code as this idea asap